Tools/Scripts/webkitpy/port/base.py

```diff
@@ -972,6 +975,25 @@ def start_web_platform_test_server(self, additional_dirs=None, number_of_servers
         Port._web_platform_test_server = web_platform_test_server.WebPlatformTestServer(self, "wptwk")
         Port._web_platform_test_server.start()
 
+        # Wait until a 5 seconds timeout happens or the HTTP server has actually finally started.
+        # https://gist.github.com/butla/2d9a4c0f35ea47b7452156c96a4e7b12
+        # Copyright (c) 2017 Michał Bultrowicz - licensed under MIT.
```

Comment:
Please put your copyright at the start of the file. We don’t want separate copyright notices on each code block.